### ISSUE TITLE
[UX] Add some text on installer explaining why User-1 is special

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#2366]
Since people sometimes don't understand that the first user on a Backdrop site is any different from any other user, I thought it might be nice to add some **concise** text to the installer, when the account is created, explaining why the first account is not simply an Administrator.  My first pass at the text was as follows:

> This account is special in that it bypasses all permission checks. Please guard these credentials safely.

But @docwilmot pointed out that...

> The new description text about bypassing permission checks is still too much for a first installer.

(I think @klonos agreed) And @docwilmot recommended, instead:

> 'This account has full administrative permissions including the ability to delete all users and content. Please guard these credentials carefully'

I then pointed out that version completely missed the point of adding the text...

> There's nothing in this wording that indicates the master account is special. I like the language `full administrative permissions` but we still need to explain that/how it's different from all the other accounts...

And instead I suggested: 

>  What if we made it more personal? Something like.. 'When you log in using this account, access restrictions set anywhere on the site will not apply to you.`

But we didn't get any specific feedback about that. Instead removed the addition of this text from that issue since it wasn't intended to start a debate there :)  But if we like, we can do that here!

**Comment by klonos** [comment]
> (I think @klonos agreed) And @docwilmot recommended, instead:

>> 'This account has full administrative permissions including the ability to delete all users and content. Please guard these credentials carefully'

Yep, I still like the text that @docwilmot suggests because it is not as "techie" as the mention of "bypassing all permission checks" in @jenlampton's text. I realize that other roles (and thus users) may be given the permission to delete users and content, but that specific user has those permissions out of the box and people should be aware of that whether they allow more users/roles to do these things or not.

If I was a beginner with Drupal/Backdrop or CMS'es in general, I would really appreciate the heads-up this message communicates. I'd be like "Yep, definitely not handing that username and password to the site owner" (which is what I currently do) 😄

**Comment by klonos** [comment]
> ...which is what I currently do

...that is I'm NOT handing it. Just to clarify.
